src/theme/NavbarItem/index.js

```diff
@@ -12,7 +12,7 @@ export default function NavbarItemWrapper(props) {
   const { pathname } = useLocation();
 
   const childProps = { ...props };
-  if (type === "docsVersionDropdown") {
+  if (type === "docsVersionDropdown" || type === "docsVersion") {
```

Comment:
This tricks the navbar into displaying the correct version for the single version navbar item. (It was already doing this for the version dropdown.)

I'd make this change in `main`, because all archived versions will want this.